plugins/inputs/prometheus/prometheus.go

```diff
@@ -77,6 +88,15 @@ func init() {
 		return &Prometheus{
 			mbCh:         make(chan PrometheusMetricBatch, 10000),
 			shutDownChan: make(chan interface{}),
+			middleware: agenthealth.NewAgentHealth(
+				zap.NewNop(),
+				&agenthealth.Config{
+					IsUsageDataEnabled:  envconfig.IsUsageDataEnabled(),
+					Stats:               &agent.StatsConfig{Operations: []string{"PutMetricData"}},
```

Comment:
Why are the stats configured and why is it PutMetricData? The only thing this should be collecting is the status codes.